config/routes.rb

```diff
@@ -2,8 +2,9 @@
   scope "(:locale)", locale: /en|vi/ do
     root "static_pages#home"
 
-    get "static_pages/home"
-    get "static_pages/help"
-    get "static_pages/about"
+    get "help" => "static_pages#help"
```

Comment:
`get "help" => "static_pages#help"` -> thành ` get "/help", to: "static_pages#help"` nhé, fix tương tự line bên dưới